pages/createpage/index.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Footer from "../../components/Footer/Footer";
+import CreateMemoryForm from "../../components/Forms/CreateMemoryForm";
+import { StyledGridWrapper, StyledMain } from "../../components/GlobalStyles";
+import Header from "../../components/Header/Header";
+
+export default function CreatePage({ sampleEvents, onAddCreateCard }) {
+  return (
+    <>
```

Comment:
your StyledGridWrapper is also a container - you don´t really need the Fragments here

Reply:
Thanks, good point